app/furniture/marketplace/order.rb

```diff
@@ -1,5 +1,24 @@
 class Marketplace
-  class Order < Checkout
+  class Order < Record
+    self.table_name = "marketplace_carts"
     self.location_parent = :marketplace
+
+    belongs_to :marketplace, inverse_of: :orders
+    delegate :space, :room, to: :marketplace
+
+    belongs_to :shopper, inverse_of: :orders
+
+    has_many :ordered_products, inverse_of: :order, foreign_key: :cart_id
+    has_many :products, through: :ordered_products, inverse_of: :orders
+
+    enum status: {
+      paid: "paid"
```

Comment:
note: I can imagine more order statuses will need to be added once customers start using the feature. `refunded`, `canceled`, and maybe some of the ones listed [here](https://support.bigcommerce.com/s/article/Order-Statuses?language=en_US).

But yeah let's Keep It Simple for now.

Reply:
Concur!